client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -5,6 +5,7 @@ import page from 'page';
 import PropTypes from 'prop-types';
 import React, { Fragment, useMemo, useCallback } from 'react';
 import { connect, useSelector, useDispatch } from 'react-redux';
+import { find, some } from 'lodash';
```

Comment:
Since both `find` and `some` are natively available on JS arrays, and because lodash imports are very expensive (as lodash heavily uses itself), could we replace these with the native versions?

Reply:
Yup, done.